site/en/docs/privacy-sandbox/videos/index.md

```diff
@@ -0,0 +1,320 @@
+description: >
+  Privacy Sandbox video library
+date: 2023-06-26
+authors:
+  - nmichell
+---
+
+If you're looking for Privacy Sandbox resources, check out the videos below. Some are recorded Privacy Sandbox office hours. You can find upcoming office hours on the [events page](/docs/privacy-sandbox/events/).
+
+{% Aside %}
+Keep in mind that while the concepts discussed in these videos are still relevant, some technical details and code will have changed.
+
+Refer to the linked documentation for the latest details.
+{% endAside %}
+
+## What is the Privacy Sandbox?
+
+{% Columns %}
+{% Column %}
+<iframe width="280" src="https://www.youtube.com/embed/NKz5oT6kXI4" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+<br>Refer to the documentation for the latest details: <a href="/docs/privacy-sandbox/overview/">What is the Privacy Sandbox?</a>
+{% endColumn %}
+{% Column %}
+<strong>May 12, 2023. Run time: 23:43</strong>
+
+Privacy Sandbox team members provide answers to common questions in this talk. After watching, you'll have a better understanding of the long-term vision, including similarities and differences between Chrome and Android implementations.
+{% endColumn %}
+{% endColumns %}
+
+{% Columns %}
+{% Column %}
+<iframe width="280" src="https://www.youtube.com/embed/gm8O8-b2B8c" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+<br>Refer to the documentation for the latest details: <a href="/docs/privacy-sandbox/overview/">What is the Privacy Sandbox?</a>
+{% endColumn %}
+{% Column %}
+<strong>May 10, 2023. Run time: 15:52</strong>
+
+Chrome is on the path to phasing out third-party cookies. Updates to cookie functionality like CHIPS and First-Party Sets enable you to restrict cross-site cookies on your site in a more secure and private manner. Learn how they can help.
+{% endColumn %}
+{% endColumns %}
+
+{% Details %}
+{% DetailsSummary %}
+### More Privacy Sandbox videos
+{% endDetailsSummary %}
+{% Columns %}
+{% Column %}
+<iframe width="280" height="158" src="https://www.youtube.com/embed/VVgtGkRtVPU" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+<br>
+Refer to the documentation for the latest details: <a href="/docs/privacy-sandbox/overview/">What is the Privacy Sandbox?</a>
+{% endColumn %}
+{% Column %}
+<strong>May 12, 2022. Run time: 16:16</strong>
+
+<strong>Latest docs:</strong> [What is the Privacy sandbox?](/docs/privacy-sandbox/overview/)
+
+At Google I/O 2022, Rowan Merewood from the Chrome team spoke about the Privacy Sandbox, the pros and cons of third-party content, and how the Privacy Sandbox addresses these issues.
```

Comment:
Where names are mentioned and there's an author profile, that would be good to link.